src/lumigo_tracer/parsers/event_parser.py

```diff
@@ -82,10 +94,58 @@ def parse(event) -> Dict:
         return new_event
 
 
+class SNSHandler(EventParseHandler):
+    @staticmethod
+    def is_supported(event) -> bool:
+        if (
+            isinstance(event, dict)
+            and event.get("Records")  # noqa
+            and event.get("Records", [{}])[0].get("EventSource") == "aws:sns"  # noqa
```

Comment:
you might have bug here. If the event is `{"Records": {"bla": "bla2"}}` (I.e. just a customer with weird event)

Reply:
I just remembered that I wrote utils for it: `safe_get`!
Just write: `safe_get(event, ['Records', 0, 'EventSource'])` 😃